zsl/constants.py

```diff
@@ -0,0 +1,8 @@
+class MimeType(object):
+    APPLICATION_JSON = 'application/json'
```

Comment:
> Content-type Content-Type or content-type 
headers are case insensitive by design and libraries should respect that, like flask e.g. `request headers['Content-type'] == request.headers['content-type']`

I'm not against using constants, but I noticed that many libraries I worked with in python, go, javascript don't use them for headers or mimetype. 

It's your call, I just think this does not feel like python.